plugins/woocommerce/client/blocks/assets/js/blocks/featured-items/with-featured-item.tsx

```diff
@@ -329,6 +389,7 @@ export const withFeaturedItem =
 				<Component
 					{ ...props }
 					backgroundImageSize={ backgroundImageSize }
+					isBgVisible={ backgroundColorVisibilityStatus }
```

Comment:
Let's rename `isBgVisible` to `backgroundColorVisibilityStatus` for consistency.